### ISSUE TITLE
oscap-docker gives many error messages when scanning a RHEL6 image

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap#474]
Hi, I tried to scan a freshly pulled RHEL6 image using the standard profile.
Although the results were produced, I got many error messages.
See the following snippet.
I use openscap-1.2.9-1.fc23.x86_64 and docker-1.10.3-24.gitf476348.fc23.x86_64

```
[root@thinkpad jcerny]# oscap-docker image rhel6 xccdf eval --profile standard --report report.html /usr/share/xml/scap/ssg/content/ssg-rhel6-xccdf.xml 
WARNING: Skipping http://www.redhat.com/security/data/oval/Red_Hat_Enterprise_Linux_6.xml file which is referenced from XCCDF content
error: unrecognized db option: "hash" ignored.
error: unrecognized db option: "tmpdir" ignored.
error: unrecognized db option: "create" ignored.
error: unrecognized db option: "mpool" ignored.
error: unrecognized db option: "perms" ignored.
error: unrecognized db option: "hash" ignored.
error: unrecognized db option: "tmpdir" ignored.
error: unrecognized db option: "create" ignored.
error: unrecognized db option: "mpool" ignored.
error: unrecognized db option: "perms" ignored.
error: unrecognized db option: "hash" ignored.
error: unrecognized db option: "tmpdir" ignored.
error: unrecognized db option: "create" ignored.
error: unrecognized db option: "mpool" ignored.
error: unrecognized db option: "perms" ignored.
error: unrecognized db option: "hash" ignored.
error: unrecognized db option: "tmpdir" ignored.
error: unrecognized db option: "create" ignored.
error: unrecognized db option: "mpool" ignored.
error: unrecognized db option: "perms" ignored.
error: unrecognized db option: "hash" ignored.
error: unrecognized db option: "tmpdir" ignored.
error: unrecognized db option: "create" ignored.
error: unrecognized db option: "mpool" ignored.
error: unrecognized db option: "perms" ignored.
error: unrecognized db option: "hash" ignored.
error: unrecognized db option: "tmpdir" ignored.
error: unrecognized db option: "create" ignored.
error: unrecognized db option: "mpool" ignored.
error: unrecognized db option: "perms" ignored.

Command: oscap xccdf eval --profile standard --report report.html /usr/share/xml/scap/ssg/content/ssg-rhel6-xccdf.xml failed!

Error was:

Command '['oscap', 'xccdf', 'eval', '--profile', 'standard', '--report', 'report.html', '/usr/share/xml/scap/ssg/content/ssg-rhel6-xccdf.xml']' returned non-zero exit status 2
```

**Comment by ybznek** [comment]
Same issue here: https://github.com/OpenSCAP/openscap/issues/425

**Comment by evgenyz** [comment]
@jan-cerny Is this bug still actual? Are we going to work on it?

**Comment by jan-cerny** [comment]
@evgenyz 

On RHEL7 which is the main system where oscap-docker command is used this bug still occurs. I checked with openscap-1.2.17-9.el7.x86_64 and docker-1.13.1-109.gitcccb291.el7_7.x86_64 and I get very similar messages.

I'm currently unable to run Docker on Fedora 31,  so I tried Podman instead. It doesn't happen to me with oscap-podman on Fedora 31, using openscap-1.3.2-2.fc31.x86_64 and podman-1.8.0-2.fc31.x86_64. I think that we might have it fixed somehow by transition to Podman.
